base.mako

```diff
@@ -68,5 +68,35 @@
             ga('create', 'UA-104682927-1', 'auto');
             ga('send', 'pageview');
         </script>
+        <script>
+(function () {
+  function copyTextToClipboard(text) {
+    var textArea = document.createElement("textarea");
+    textArea.value = text;
+    textArea.style.top = "0";
+    textArea.style.left = "0";
+    textArea.style.position = "fixed";
+    document.body.appendChild(textArea);
+    textArea.focus();
+    textArea.select();
+    document.execCommand('copy');
+    document.body.removeChild(textArea);
+  }
+  var codeBlockElements = document.querySelectorAll('div#advanced ul')[8].getElementsByClassName("highlight");
```

Comment:
this seems very fragile -- perhaps we can add a specific class to the section containing this?

Reply:
- agree
- first thought is to add some `html` to the template and add whatever needed (id or class), do you think there is a better way doing it?

Reply:
we already do some custom stuff to render some of the code blocks -- could probably just adjust that [here](https://github.com/pre-commit/pre-commit.github.io/blob/bff157f8a1f8651e1b9089280c8eef36648ffaca/template_lib.py#L102-L108)

maybe something like

```markdown

    ```markdown  # copyable
    ...
    ```
```

Reply:
WIP